### PR TITLE
Emit event when TrueFiOracle credit scores are changed

### PR DESCRIPTION
**Pull request by skalermo**

### CODE REVIEW

File: test/truefi2/TrueFiCreditOracle.test.ts [hunk 1]
```diff
@@ -41,24 +41,37 @@ describe('TrueFiCreditOracle', () => {
...
     })
 
-    it('only manager can set scores', async () => {
+    it('reverts if caller is not the manager', async () => {
       await expect(oracle.connect(owner).setScore(firstAccount.address, firstScore))
         .to.be.revertedWith('TrueFiCreditOracle: Caller is not the manager')
     })
 
-    it('score is set correctly for account', async () => {
+    it('sets correct score for the account', async () => {
```
Comment (MiksuJak) [inline test/truefi2/TrueFiCreditOracle.test.ts:57]: What was wrong with this and previous test names?

File: test/truefi2/TrueFiCreditOracle.test.ts [hunk 2]
```diff
@@ -41,24 +41,37 @@ describe('TrueFiCreditOracle', () => {
...
+        .to.emit(oracle, 'ScoreChanged')
+        .withArgs(firstAccount.address, secondScore)
+    })
+  })
+
+  describe('getScore', () => {
+    const firstScore = 100
+    const secondScore = 200
+
+    it('gets correct score after it changed', async () => {
```
Comment (MiksuJak) [inline test/truefi2/TrueFiCreditOracle.test.ts:72]:
```suggestion
    it('gets correct score after it has changed', async () => {
```

File: test/truefi2/TrueFiCreditOracle.test.ts (hunk 1 above)
Comment (skalermo) [reply test/truefi2/TrueFiCreditOracle.test.ts:57]: My bad, I'm so used to this naming style I changed it subconsciously. I can change it back if you like.

**Review by MiksuJak — APPROVED** [review]

**Review by yuchenlintt — APPROVED** [review]
Looks good!

**Pull request merged**